polygon/rest.go

```diff
@@ -0,0 +1,205 @@
+package polygon
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+	try "gopkg.in/matryer/try.v1"
+)
+
+const (
+	aggURL    = "%v/v1/historic/agg/%v/%v"
+	tradesURL = "%v/v1/historic/trades/%v/%v"
+	quotesURL = "%v/v1/historic/quotes/%v/%v"
+)
+
+var (
+	base string
+	get  = func(u *url.URL) (*http.Response, error) {
```

Comment:
Weird space here

Reply:
what do you mean weird space? it's go fmt...